tests/event_config.at

```diff
@@ -58,3 +58,130 @@ TS_MAIN
 }
 TS_RETURN_MAIN
 ]])
+
+## ------------------------ ##
+## get_options_with_err_msg ##
+## ------------------------ ##
+
+AT_TESTFUN([get_options_with_err_msg],
+[[
+
+#include "testsuite.h"
+#include "internal_libreport.h"
+
+#undef EVENTS_DIR
+#define EVENTS_DIR "../../conf/bugtester/"
+#undef EVENTS_CONF_DIR
+#define EVENTS_CONF_FIR "../../conf/bugtester"
+
+static int cmp_event_option_name_with_string(gconstpointer a, gconstpointer b)
+{
+    const event_option_t *evopt = a;
+    return !evopt->eo_name || strcmp(evopt->eo_name, (char *)b) != 0;
+}
+
+
+static void load_config_files(const char *dir_path)
```

Comment:
You are right. It is not needed. I will replace the loading of the files with manually created event config with option list.